api/src/modules/impact/impact.controller.ts

```diff
@@ -24,4 +27,36 @@ export class ImpactController {
     );
     return { data: impactTable };
   }
+
+  @ApiOperation({ description: 'Return materials loaded by a user' })
+  @ApiOkResponse({
+    type: Material,
+    isArray: true,
+  })
+  @Get('materials')
+  async getMaterialTreeForImpact(): Promise<Material[]> {
+    return this.impactService.getMaterialTreeForImpact();
```

Comment:
so the impact module has this controller and this endpoint, which is named after material, but then the service uses the sourcing location service, and the material service, and then it all is serialized through a material module serializer.... not sure I like it...